### PR TITLE
docs: removal of unwanted extra closure of achor tag

### PR DESCRIPTION
**Pull request by dhairyathedev**
### Check off the following:

- [] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/rest/guides/basics-of-authentication.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Next, in _views/index.erb_, paste this content:
     </p>
     <p>
       We're going to now talk to the GitHub API. Ready?
-      <a href="https://github.com/login/oauth/authorize?scope=user:email&client_id=<%= client_id %>">Click here</a> to begin!</a>
```
Comment (Boobie84) [inline content/rest/guides/basics-of-authentication.md:79]: content/rest/guides/basics-of-authentication.md

Comment (codewithsnowbit) [reply content/rest/guides/basics-of-authentication.md:79]: ???

Comment (codewithsnowbit) [reply content/rest/guides/basics-of-authentication.md:79]: ???

**Review by ramyaparimi — APPROVED** [review]
@codewithsnowbit 
Thanks so much for opening a PR with this fix! I'll get this merged once tests are passing 🍏

**Comment by dr-suhadalishaheed** [comment]
H #
be save

**Comment by codewithsnowbit** [comment]
> @codewithsnowbit Thanks so much for opening a PR with this fix! I'll get this merged once tests are passing 🍏

You're welcome @ramyaparimi

**Pull request merged**